### PR TITLE
update proposal links after transfer to tc39 org

### PR DESCRIPTION
**Pull request by michaelficarra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This list contains only stage 1 proposals and higher that have not yet been with
 |   | [`Promise.prototype.finally`](https://github.com/tc39/proposal-promise-finally)                             | Jordan Harband                     | 3 |
 |   | [BigInt](https://github.com/tc39/proposal-bigint)                                                           | Daniel Ehrenberg                   | 3 |
 |   | [Class Fields](https://github.com/tc39/proposal-class-fields)                                               | Daniel Ehrenberg, Jeff Morrison    | 3 |
-|   | [Optional catch binding](https://michaelficarra.github.io/optional-catch-binding-proposal/)                 | Michael Ficarra                    | 3 |
+|   | [Optional catch binding](https://tc39.github.io/proposal-optional-catch-binding/)                           | Michael Ficarra                    | 3 |
```
Comment (ljharb) [inline README.md]: it'd probably be better to link to the repo rather than the rendered spec link

File: README.md [hunk 2]
```diff
@@ -53,12 +53,12 @@ This list contains only stage 1 proposals and higher that have not yet been with
...
 |   | [Atomics.waitNonblocking](https://github.com/lars-t-hansen/moz-sandbox/blob/master/sab/waitNonblocking.md)  | Shu-yu Guo, Lars Hansen            | 1 |
-|   | [`Symbol.prototype.description`](https://github.com/michaelficarra/Symbol-description-proposal/)            | Michael Ficarra                    | 1 |
+|   | [`Symbol.prototype.description`](https://github.com/tc39/proposal-Symbol-description/)                       | Michael Ficarra                    | 1 |
 |   | change Number.parseInt/parseFloat to not coerce null/undefined/NaN (repo link TBD)                          | Brendan Eich                       | 1 |
 |   | [Array.prototype.flat{Map,ten}](https://github.com/tc39/proposal-flatMap)                                   | Brian Terlson, Michael Ficarra     | 1 |
 |   | [Throw expressions](https://github.com/rbuckton/proposal-throw-expressions)                                 | Rob Buckton                        | 1 |
 
-🚀 means the champion thinks it's ready to advance but has not yet presented to the committee.
+ means the champion thinks it's ready to advance but has not yet presented to the committee.
```
Comment (ljharb) [inline README.md]: looks like you may have lost the rocket emoji here :-/

Comment (ljharb) [inline README.md]: no worries tho, I'll fix it and merge

Comment (michaelficarra) [inline README.md]: Blame GitHub's online editor. Fixed in vim.

**Pull request merged**